### PR TITLE
Measure per-architecture sizes for firebase crashlytics ndk.

### PR DESCRIPTION
**Pull request by yifanyang**

### CODE REVIEW

**Comment by yifanyang** [comment]
@kmandrika Hey Konstantin, upon our discussion last week, I've adjusted our size measurement setup. Now the measurement for `firebase-crashlytics-ndk` are collected per architecture:


* #### `firebase-crashlytics-ndk`

  | Type | Base (82b02af3) | Head (81defe97) | Diff |
  | --- |:---:|:---:|:---:|
  | apk (aggressive / arm64-v8a) | ? | 1.34 MB | ? (?) |
  | apk (aggressive / armeabi-v7a) | ? | 839 kB | ? (?) |
  | apk (aggressive / x86) | ? | 1.38 MB | ? (?) |
  | apk (aggressive / x86_64) | ? | 1.44 MB | ? (?) |
  | apk (release / arm64-v8a) | ? | 1.97 MB | ? (?) |
  | apk (release / armeabi-v7a) | ? | 1.47 MB | ? (?) |
  | apk (release / x86) | ? | 2.01 MB | ? (?) |
  | apk (release / x86_64) | ? | 2.08 MB | ? (?) |


It's also shown in the above PR size report.

Does the way of constructing the app and the way for displaying in PR look good to you?

File: apk-size/app/configure.gradle [hunk 1]
```diff
@@ -26,9 +26,30 @@ if (project.hasProperty('sdks')) {
...
             dependencies.add("${artifactId}Implementation", sdk)
         }
+
+        // Ignore debug builds as measurements are not reproducible due to disabled ZipAlign.
+        // Ignore per-architecture builds for all products except for firebase-crashlytics-ndk.
+        variantFilter { variant ->
+            def buildType = variant.buildType.name
+            def flavors = variant.flavors*.name
+            if (buildType == 'debug'
+                    || 'firebase-crashlytics-ndk' in flavors && 'universal' in flavors
```
Comment (vkryachko) [inline apk-size/app/configure.gradle:40]: Is this supposed to be grouped together lilke?
```suggestion
                    || ('firebase-crashlytics-ndk' in flavors && 'universal' in flavors)
```

Comment (yifanyang) [reply apk-size/app/configure.gradle]: Yes. Thanks for catching that!

**Review by vkryachko — APPROVED** [review]

**Pull request merged**